src/Container/AgFeatureGrid/AgFeatureGrid.jsx

```diff
@@ -0,0 +1,1019 @@
+     * @type {ol.Map}
+     */
+    map: PropTypes.instanceOf(OlMap),
+
+    /**
+     * A list of attribute names to hide in the table.
+     * @type {Array}
+     */
+    attributeBlacklist: PropTypes.arrayOf(PropTypes.string),
+
+    /**
+     * Optional callback function, that will be called on rowclick.
+     * @type {Function}
+     */
+    onRowClick: PropTypes.func,
+
+    /**
+     * Optional callback function, that will be called on rowmouseover.
+     * @type {Function}
+     */
+    onRowMouseOver: PropTypes.func,
+
+    /**
+     * Optional callback function, that will be called on rowmouseout.
+     * @type {Function}
+     */
+    onRowMouseOut: PropTypes.func,
+
+    /**
+     * Optional callback function, that will be called if the selection changes.
+     * @type {Function}
+     */
+    onRowSelectionChange: PropTypes.func,
+
+    /**
+     * Whether the map should center on the current feature's extent on init or
+     * not.
+     * @type {Boolean}
+     */
+    zoomToExtent: PropTypes.bool,
+
+    /**
+     * Whether features should be selectable via a map click or not. If you want
+     * to enable/disable a checkbox, please set checkboxSelection on a column
+     * in columnDefs.
+     *
+     * @type {Boolean}
+     */
+    selectable: PropTypes.bool,
+
+    /**
+     * The default style to apply to the features.
+     * @type {ol.Style|ol.FeatureStyleFunction}
+     */
+    featureStyle: PropTypes.oneOfType([
+      PropTypes.instanceOf(OlStyle),
+      PropTypes.func
+    ]),
+
+    /**
+     * The highlight style to apply to the features.
+     * @type {ol.Style|ol.FeatureStyleFunction}
+     */
+    highlightStyle: PropTypes.oneOfType([
+      PropTypes.instanceOf(OlStyle),
+      PropTypes.func
+    ]),
+
+    /**
+     * The select style to apply to the features.
+     * @type {ol.Style|ol.FeatureStyleFunction}
+     */
+    selectStyle: PropTypes.oneOfType([
+      PropTypes.instanceOf(OlStyle),
+      PropTypes.func
+    ]),
+
+    /**
+     * The name of the vector layer presenting the features in the grid.
+     * @type {String}
+     */
+    layerName: PropTypes.string,
+
+    /**
+     * Custom column definitions to apply to the given column (mapping via key).
+     * See https://ant.design/components/table/#Column.
+     * @type {Object}
+     */
+    columnDefs: PropTypes.object,
+
+    /**
+     * The children to render.
+     * @type {Element}
+     */
+    children: PropTypes.element,
+
+    /**
+     * A Function that creates the rowkey from the given feature.
+     * Receives the feature as property.
+     * Default is: feature => feature.ol_uid
+     *
+     * @type {Function}
+     */
+    keyFunction: PropTypes.func,
+
+    /**
+     * A Function that is called once the grid is ready.
+     * @type {Function}
+     */
+    onGridIsReady: PropTypes.func
+  };
+
+  /**
+   * The default properties.
+   * @type {Object}
+   */
+  static defaultProps = {
+    theme: 'ag-theme-balham',
```

Comment:
We should mention that the specific CSS for this theme should be loaded, right?

Reply:
It is already mentioned in the documentation of the corresponding prop, see [here](https://github.com/terrestris/react-geo/pull/588/files#diff-8149a62228de5b566a4a4a816147b181R105).

Reply:
I know, I was specifically talking about things we need for the default theme.

Reply:
The CSS for the default theme is imported [here](https://github.com/terrestris/react-geo/pull/588/files#diff-8149a62228de5b566a4a4a816147b181R25).